spec/support/features/in_person_helper.rb

```diff
@@ -60,7 +61,7 @@ def fill_out_address_form_ok(double_address_verification: false, same_address_as
     fill_in t('idv.form.zipcode'), with: same_address_as_id ? GOOD_STATE_ID_ZIPCODE : GOOD_ZIPCODE
     if same_address_as_id
       select GOOD_STATE_ID_JURISDICTION,
-             from: t('in_person_proofing.form.state_id.state_id_jurisdiction')
+             from: t('in_person_proofing.form.state_id.state_id_state')
```

Comment:
I'm confused by the `select GOOD_STATE_ID_JURISDICTION` but then selecting it from the state_id_state input

Reply:
Thanks for catching this, it was wrong and exposed a weakness in testing.  (The state id address and residential address previously had the same state, which I changed in the constants file.)